### ISSUE TITLE
XDMoD 10.0.0 Prompt

### ISSUE THREAD

**Issue by swansun** [ubccr/hpc-toolset-tutorial#112]
Since the XDMoD 10 release, the xdmod-setup prompts about the new version, then fails.

```
$ docker-compose logs xdmod
Attaching to xdmod
xdmod        | ---> Starting SSSD on xdmod ...
xdmod        | ---> Starting sshd on xdmod...
xdmod        | ---> Starting the MUNGE Authentication service (munged) on xdmod ...
xdmod        | ---> Starting sshd on xdmod...
xdmod        | ERROR 2003 (HY000): Can't connect to MySQL server on 'mysql' (111)
xdmod        | -- Waiting for database to become active ...
xdmod        | ERROR 2003 (HY000): Can't connect to MySQL server on 'mysql' (111)
xdmod        | -- Waiting for database to become active ...
xdmod        | ERROR 2003 (HY000): Can't connect to MySQL server on 'mysql' (111)
xdmod        | -- Waiting for database to become active ...
xdmod        | ---> Open XDMoD Setup: SSO...
xdmod        | ---> Open XDMoD Setup: start
xdmod        | spawn xdmod-setup
xdmod        | You are currently using Open XDMoD 9.5.0, but a newer version
xdmod        | (10.0.0) is available.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no] 1
xdmod        | 
xdmod        | '1' is not a valid option.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no]
xdmod        | Failed to get prompt
xdmod        | ---> Open XDMoD Setup: hpc resource
xdmod        | spawn xdmod-setup
xdmod        | You are currently using Open XDMoD 9.5.0, but a newer version
xdmod        | (10.0.0) is available.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no] 4
xdmod        | 
xdmod        | '4' is not a valid option.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no] 1
xdmod        | 
xdmod        | '1' is not a valid option.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no]
xdmod        | Failed to get prompt
xdmod        | ---> Open XDMoD Setup: finish
xdmod        | spawn xdmod-setup
xdmod        | You are currently using Open XDMoD 9.5.0, but a newer version
xdmod        | (10.0.0) is available.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no] 5
xdmod        | 
xdmod        | '5' is not a valid option.
xdmod        | 
xdmod        | Do you want to continue (yes, no)? [no]
xdmod        | Failed to get prompt
xdmod        | Open XDMoD Import: Hierarchy
xdmod        | (2022-04-15 21:26:51): [be[default]] [sysdb_get_real_name] (0x0040): Cannot find user [xdmod@default] in cache
xdmod        | (2022-04-15 21:26:51): [be[default]] [sysdb_get_real_name] (0x0040): Cannot find user [xdmod@default] in cache
xdmod        | (2022-04-15 21:26:51): [be[default]] [sysdb_get_real_name] (0x0040): Cannot find user [xdmod@default] in cache
xdmod        | (2022-04-15 21:26:51): [be[default]] [sysdb_get_real_name] (0x0040): Cannot find user [xdmod@default] in cache
xdmod        | No entry for terminal type "unknown";
xdmod        | using dumb terminal settings.
xdmod        | SQLSTATE[HY000] [2002] Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2)
xdmod        | #0 /usr/share/xdmod/classes/CCR/DB/PDODB.php(88): PDO->__construct('mysql:host=loca...', 'xdmod', '')
xdmod        | #1 /usr/share/xdmod/classes/CCR/DB.php(111): CCR\DB\PDODB->connect()
xdmod        | #2 /usr/share/xdmod/classes/CCR/CCRDBHandler.php(55): CCR\DB::factory('logger')
xdmod        | #3 /usr/share/xdmod/classes/CCR/Log.php(288): CCR\CCRDBHandler->__construct(NULL, NULL, NULL, 200)
xdmod        | #4 [internal function]: CCR\Log::getDbHandler('xdmod-import-cs...', Array)
xdmod        | #5 /usr/share/xdmod/classes/CCR/Log.php(192): call_user_func(Array, 'xdmod-import-cs...', Array)
xdmod        | #6 /usr/share/xdmod/classes/CCR/Log.php(113): CCR\Log::getLogger('xdmod-import-cs...', Array)
xdmod        | #7 /usr/bin/xdmod-import-csv(133): CCR\Log::factory('xdmod-import-cs...', Array)
xdmod        | #8 /usr/bin/xdmod-import-csv(27): main()
xdmod        | #9 {main}
```

**Comment by aebruno** [comment]
@swansun Thanks for reporting this. This should be resolved in the next update for PEARC22.

**Comment by aebruno** [comment]
Fixed in #127
